harpy/_validations.py

```
@@ -351,7 +351,11 @@ def validate_regions(regioninput, genome):
 def check_fasta(genofile):
     """perform validations on fasta file for extensions and file contents"""
     # validate fasta file contents
-    opener = gzip.open if is_gzip(genofile) else open
+    try:
+        opener = gzip.open if is_gzip(genofile) else open
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)